### DISCUSSION TITLE
Retore backup to new Google Workspace

### DISCUSSION THREAD

**Discussion by chrisyoungcooks** [GAM-team/got-your-back#443] [Q&A]
I'm struggling to restore a backup from an email address on one Google Workspace (that's going away soon) and migrating the emails to a new Google Workspace. When I attempt to run the backup with the new email address having a different domain for the new Workspace, I run into the error "Access blocked: GYB can only be used within its organization" and I can't seem to find a way to give the current GYB Project access to a second Google Workspace. Any help would be appreciated.

**Comment by dab15** [comment]
You need to remove (rename) the original OAUTH json and SECRETS files that you created with your Google Workspace account, and go through the project setup steps again (this time with the new email).  Setup is pretty well outlined in the project wiki: https://github.com/GAM-team/got-your-back/wiki#general and then restoration (migration) to a new account is detailed here: https://github.com/GAM-team/got-your-back/wiki#performing-a-restore.  Does this work? Did for me with absolutely no prior experience with GYB. Good luck - it's stressful to have to do this!
